doctrine/annotations/1.0/config/routes/annotations.yaml

```diff
@@ -0,0 +1,3 @@
+controllers:
+    resource: ../src/Controller/
```

Comment:
since this is relative to `%kernel.root_dir%` it should be just `Controller/`

Reply:
I think it's relative to the entry point (`public/index.php` or `bin/console`) where the kernel is instantiate.